express-js-bootstrap-graphql/test/util/usersQueries.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const chai = require('chai'),
+  server = require('../../app');
+
+const userOne = {
+  name: 'sarahi',
+  lastName: 'torres',
+  email: 'sarahi12hg@wolox.com',
+  password: 'woloxwoloA1520'
+};
+
+const mutationCreateUser = {
+  query:
+    '\nmutation{\ncreateUser(data:{name:"sarahi",lastName:"Torres", email:"sarahitorres@wolox.jp",password:"AquiEAst156xcZ"})\n}'
+};
+exports.userOne = userOne;
```

Comment:
instead of doing all this reassignment you could add
```
module.exports = {
  userOne,
  mutationCreateUser
}
```
and it would be the same as doing
```
exports.userOne = userOne;
exports.mutationCreateUser = mutationCreateUser'
```
so its a little less verbose. Also some of you naming conventions seem to be backwards, in english you wouldnt really say `mutationCreateUser` but rather `createUserMutation`